server/index.js

```diff
@@ -2,11 +2,77 @@ require('dotenv/config');
 const express = require('express');
 const errorMiddleware = require('./error-middleware');
 const staticMiddleware = require('./static-middleware');
+const ClientError = require('./client-error');
+const pg = require('pg');
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
 
 const app = express();
 
+app.use(express.json());
+
 app.use(staticMiddleware);
 
+app.get('/api/get-posts', (req, res, next) => {
+  const sql = `
+  select *
+  from "postedGames"
+  `;
+
+  db.query(sql)
+    .then(result => {
+      res.json(result.rows);
+    })
+    .catch(err => next(err));
+});
+
+app.post('/api/create-post', (req, res, next) => {
```

Comment:
I still don't really understand how a post is different from a game. Also, the convention these days is 

```js
app.post('/api/things', createThingCallback)

app.delete('/api/things/:thingId', deleteThingCallback)
```

We use a collection name and tack on `/:thingId` at the end to target a specific item in the collection.